cmd/localstack/filenotify/filenotify.go

```diff
@@ -32,9 +32,9 @@ func shouldUseEventWatcher() bool {
 	err := unix.Uname(&utsname)
 	release := strings.TrimRight(string(utsname.Release[:]), "\x00")
 	log.Println("Release detected: ", release)
-	// cheap check if we are in Docker desktop or not.
+	// cheap check if we are in Docker desktop for Windows (WSL2) or not.
 	// We could also inspect the mounts, but that would be more complicated and needs more parsing
-	return err == nil && !(strings.Contains(release, "linuxkit") || strings.Contains(release, "WSL2"))
+	return err == nil && !(strings.Contains(release, "WSL2"))
```

Comment:
in my macos I saw above log to print `Release detected:  6.4.16-linuxkit` so that's why we removed `linuxkit` - I wonder if this can break any other system such as windows if using linuxkit but it feels safe to me

Reply:
This is here on purpose, as inotify did not work with docker desktop at the time of implementation of this. 
Maybe this changed with the default usage of virtioFS, but this would still be a problem for people still on grpc-fuse.
I will test this more thoroughly on our macos machine tomorrow, I just wanted to provide some context here.

Even if it works with virtioFS, but not grpc, we need some mechanism to enable it in one but not the other.

Reply:
thank you! 

I confirm it works for me with both VirtioFS and gRPC FUSE in my macosx (Apple M2 Pro) and with Docker Desktop 4.25.0 (126437)

hope it works for you too

Reply:
perhaps 1 year ago when the code was written it was not working as expected, but it doesn't seem to be the case with the last versions of Docker Desktop

Reply:
Hi @javiertejero :) This looks exactly what I need to solve a similar issue. How can I apply this in my machine?

Reply:
@uriyakadmon first compile this package, you can use the `Makefile` in the repo, then you can replace the binary in the localstack but it requires some hacks, and it depends whether you use docker-compose or not... using docker-compose you can replace the old binary with the new in the `volume` folder ... and that would solve the problem on that machine

Reply:
That's exactly what I ended up doing. Thank you!